api/src/test/java/org/pmiops/workbench/api/ClusterControllerTest.java

```diff
@@ -148,14 +144,19 @@ Random random() {
 
   @Captor private ArgumentCaptor<Map<String, String>> mapCaptor;
 
-  @Autowired ClusterAuditor clusterAuditor;
-  @Autowired LeonardoNotebooksClient notebookService;
-  @Autowired FireCloudService fireCloudService;
+  @MockBean AdminActionHistoryDao adminActionHistoryDao;
```

Comment:
nit: I like to name mocked members with a `mock` prefix, especially for large test classes.